server/modules/record/service/recordService.js

```diff
@@ -159,22 +113,24 @@ const persistNode = async (user, surveyId, node, file) => {
 
     await FileManager.insertFile(surveyId, fileObj)
   }
-  const thread = _getOrCreatedRecordThread(user, surveyId, Node.getRecordUuid(node), false, true)
+  const thread = RecordServiceThreads.getOrCreatedRecordThread(user, surveyId, Node.getRecordUuid(node), false, true)
   thread.postMessage({ type: recordThreadMessageTypes.nodePersist, node, user })
 }
 
 const deleteNode = (user, surveyId, recordUuid, nodeUuid) => {
-  const thread = _getOrCreatedRecordThread(user, surveyId, recordUuid, false, true)
+  const thread = RecordServiceThreads.getOrCreatedRecordThread(user, surveyId, recordUuid, false, true)
   thread.postMessage({ type: recordThreadMessageTypes.nodeDelete, nodeUuid, user })
 }
 
 const deleteRecordsPreview = async () => {
   const surveyIds = await SurveyManager.fetchAllSurveyIds()
-  await Promise.all(
+  const nDeletedRecordsBySurvey = await Promise.all(
```

Comment:
why this variable is called nDeletedRecordsBySurvey, i cannot understand what n prefix is.

Reply:
It stands for number. renamed to countDeletedRecordsBySurvey